crml/staking/src/lib.rs

```diff
@@ -290,81 +291,118 @@ pub trait WeightInfo {
 	fn bond() -> Weight;
 	fn bond_extra() -> Weight;
 	fn unbond() -> Weight;
-	fn rebond() -> Weight;
+	fn rebond(l: u32) -> Weight;
 	fn reap_stash() -> Weight;
 	fn withdraw_unbonded() -> Weight;
 	fn validate() -> Weight;
-	fn nominate() -> Weight;
+	fn nominate(n: u32) -> Weight;
 	fn chill() -> Weight;
 	fn set_payee() -> Weight;
 	fn set_controller() -> Weight;
 	fn set_validator_count() -> Weight;
 	fn force_no_eras() -> Weight;
 	fn force_new_era() -> Weight;
 	fn set_minimum_bond() -> Weight;
-	fn set_invulnerables() -> Weight;
+	fn set_invulnerables(v: u32) -> Weight;
 	fn force_unstake() -> Weight;
 	fn force_new_era_always() -> Weight;
-	fn cancel_deferred_slash() -> Weight;
+	fn cancel_deferred_slash(s: u32) -> Weight;
 }
 
 impl WeightInfo for () {
```

Comment:
If you just remove this implementation here, then you don't need to copy paste the default implementation from Plug to here

Reply:
Yup please use `mod default_weights` and update the implementation there. this one can be removed